package.json

```diff
@@ -17,11 +17,13 @@
     "@actions/github": "^5.0.3",
     "@octokit/plugin-throttling": "^4.1.0",
     "@octokit/webhooks-types": "^6.2.2",
-    "eipw-lint-js": "^0.1.5"
+    "eipw-lint-js": "^0.1.5",
+    "tr46": "^3.0.0"
```

Comment:
Note: manually updated due to the fact that the previous version contained syntax errors (!!!)